src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -702,6 +702,16 @@ def get_all_realms_info(self):
         except NoRgwDaemonsException as e:
             raise DashboardException(e, http_status_code=404, component='rgw')
 
+    @allow_empty_body
+    # pylint: disable=W0613
+    def set(self, realm_name, default, new_realm_name, daemon_name=None):
+        try:
+            instance = RgwClient.admin_instance()
+            result = instance.edit_realm(realm_name, default, new_realm_name)
```

Comment:
no need to disable unused variable if you want to leave `daemon_name`. 
`RgwClient.admin_instance(daemon_name)`